### PR TITLE
[ PBI-7304] Add initiator module and 6.9 SDKs

### PR DESCRIPTION
**Pull request by PoornimaApptentive**
https://alchemer.atlassian.net/browse/PBI-7304

### CODE REVIEW

File: ApptentiveKit.Android/ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.csproj [hunk 1]
```diff
@@ -26,18 +26,18 @@
     <PackageReference Include="Xamarin.AndroidX.Fragment" Version="1.6.2.2" />
 	  <PackageReference Include="Xamarin.Google.Android.Play.Review" Version="2.0.1.7" />
 	  <PackageReference Include="Xamarin.Google.Android.Play.Review.Ktx" Version="2.0.1.7"/>
-    <PackageReference Include="ApptentiveMessageCenter" Version="6.8.0"/>
+    <!-- <PackageReference Include="ApptentiveMessageCenter" Version="6.8.0"/>
```
Comment (frankus) [inline ApptentiveKit.Android/ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.csproj]: Should this still be commented out?

File: Plugin.Maui.Apptentive.Sample/Platforms/Android/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.apptentive.xolofacts.latest" android:versionCode="19" android:versionName="4.1">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.apptentive.maui.latest.sample" android:versionCode="1" android:versionName="1.0">
```
Comment (frankus) [inline Plugin.Maui.Apptentive.Sample/Platforms/Android/AndroidManifest.xml:2]: Good catch!

File: Plugin.Maui.Apptentive/Plugin.Maui.Apptentive.csproj [hunk 3]
```diff
@@ -52,8 +52,8 @@
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0-android'">
 	 <PackageReference Include="ApptentiveFeedback" Version="6.8.0" />
-		<!-- For development:
-		<ProjectReference Include="..\ApptentiveKit.Android\ApptentiveCore\ApptentiveCore\ApptentiveCore.csproj" />
+		<!-- For development: -->
+		<!-- <ProjectReference Include="..\ApptentiveKit.Android\ApptentiveCore\ApptentiveCore\ApptentiveCore.csproj" />
```
Comment (frankus) [inline Plugin.Maui.Apptentive/Plugin.Maui.Apptentive.csproj:56]: Does this also still need to be commented out?

**Review by frankus — APPROVED** [review]
Looks good with a couple of questions.

File: ApptentiveKit.Android/ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.csproj (hunk 1 above)
Comment (PoornimaApptentive) [reply ApptentiveKit.Android/ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.csproj]: Probably not. I am going to uncomment. This is to ensure the locally built packages are used. Thanks for catching!

File: Plugin.Maui.Apptentive/Plugin.Maui.Apptentive.csproj (hunk 3 above)
Comment (PoornimaApptentive) [reply Plugin.Maui.Apptentive/Plugin.Maui.Apptentive.csproj:56]: Yes. This is only for development (testing purpose) as per Luqmaan. I will make a note to check with him before release.

**Review by frankus — APPROVED** [review]

**Review by frankus — APPROVED** [review]

**Pull request merged**